### DISCUSSION TITLE
Support yarn npm publish for yarn workspace repo

### DISCUSSION THREAD

**Discussion by taeyoon0137** [ghiscoding/lerna-lite#211] [Q&A] [answered]
Hi, I'm trying to use lerna-lite with existing yarn workspace monorepo.

I tried `yarn lerna publish` command but i guess it tries to publish packages via `npm publish` command, not `yarn npm publish`.
It causes license error(cause I've only defined scope for yarn via `.yarnrc.yml`).

I know it can be fixed by config new `.npmrc`,
but can I use `yarn npm publish` instead of `npm publish` in `yarn lerna publish` command?

Thanks for everyone's kindness :)

**Comment by ghiscoding** [comment, marked as answer]
Just a guess but you might be missing `"npmClient": "yarn"` in your `lerna.json` config, you can get more info in the [`lerna.json`](https://github.com/ghiscoding/lerna-lite/wiki/lerna.json#lernajson) Wiki.
